WordPress/src/main/java/org/wordpress/android/ui/accounts/NewUserFragment.java

```diff
@@ -397,10 +411,15 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
             @Override
             public void afterTextChanged(Editable s) {
+                String lowerCase = s.toString().toLowerCase();
```

Comment:
Seems that the same code is repeated at `line 391` and in `NewBlogFragment`. Would be possible to create a common method and call it from these 3 different places?